bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -153,7 +153,7 @@ metadata:
       Cloud Provider,Developer Tools,Modernization & Migration,OpenShift Optional,Storage
     certified: "false"
     containerImage: quay.io/konveyor/oadp-operator:oadp-1.4
-    createdAt: "2024-05-29T17:22:16Z"
```

Comment:
this change is right?

Reply:
@mateusoliveira43 This file was created from `make bundle` -- I'm not sure why `createdAt` was removed. I just tried again in my checkout, removed the file and re-ran make bundle and it's the same.

@shubham-pampattiwar @kaovilai Any idea why this would happen?